docs/operate/get-started/setup-micro.md

```diff
@@ -43,6 +43,10 @@ To quickly try out Viam for microcontrollers, you can use the pre-built binary `
 
 ## Quickstart
 
+{{% hiddencontent %}}
+Viam provides installers to flash your ESP32 using macOS, Linux x86_64, or Linux ARM64 (AArch64).
```

Comment:
[nit] if we're going to call out the architecture for Linux installs, should we also call it out for macOS? afaik, quite a few Mac models with x86_64 processors are still actively supported and could easily be used by customers. Might be cleaner to say something like:

```suggestion
Viam provides installers to flash an ESP32 from macOS or Linux running on the x86_64 or ARM64 (AArch64) processor architectures.
```

It does feel a little dirty to conflate Apple Silicon with all other ARM64 architectures, but for the purposes of x86/arm disambiguation here I suspect it's OK.

Reply:
The app just says this:
![image](https://github.com/user-attachments/assets/223bd131-e74a-4650-af10-30e600750210)
We haven't tended to distinguish between mac architectures. Should we?